hell/adventcalendar/2025/35/index.md

```diff
@@ -1,35 +1,102 @@
 
-Some text.
+In the HTML Living Standard there are explicit considerations for what are called [non-conforming features](https://html.spec.whatwg.org/multipage/obsolete.html#non-conforming-features). To quote the standard, "Elements in the following list are entirely obsolete, and must not be used by authors". Thus, I shall issue the disclaimer not to use these lest WHATWG show up at my door.
 
-## Heading
+`<marquee>` is perhaps the most famous of all, second only to `<blink>`, which was never standardised. Daniela Kubesch [wrote about the `marquee` tag and implementing it in a modern way](https://www.htmhell.dev/adventcalendar/2022/15/) previously, so I'll leave that to them.
 
-Some text.
+But there are far more obscure tags which are perhaps less visually dazzling but equally or even more interesting. If you're younger, this might very well be your introduction to them. If you're older, this still might be an introduction, but also possibly a trip down memory lane or a flashback to the horrors of the first browser war. It depends.
 
+## bgsound
 
-<p class="highlight"><strong>Note:</strong> Some text.</p>
+`<bgsound>` was a way to play sound in the background, because everyone loves web pages making sounds they didn't ask for at them. It was not standardised, and exclusively part of Internet Explorer. The `<audio>` tag is the much more courteous modern equivalent.
 
-```html
-<h1> 
-  <a href="/">
-    Hello World
-  </a>
-</h1>
-```
+I find it endlessly humorous that the old HTML Wiki's [example section for the tag](https://www.w3.org/html/wiki/Elements/bgsound#Examples) simply states, 'No, really. Don't use it.'. Many of the old and obscure elements share the same detail.
+
+## Framesets
+
+In the olden days, people used 'frames'. Not iframes mind you, just frames. You can think of their usage vaguely like the HTML imports we have now... Wait, what do you mean we still don't have HTML imports?
+
+I digress. This was long before single-page applications (SPAs) or anything of that nature, but were widely used as a way to aid navigation without needing to reload the page. You could have a navigation frame that persisted while the content frame changed.
+
+`<frame>` is more or less the same as our modern `<iframe>`, though we use iframes much more for embedding than page layout. `<frameset>` holds these frames, so you could write something like `<frameset cols="50%, 50%">` to have two frames next to each other.
+
+The frameset approach had quite a few problems. Accessibility was really poor, and because everything was a single page, it could be hard to link to a specific section. ~~Thankfully we've come a long way since then, and developers take great care with accessibility and ensuring their SPAs respect the web platform.~~
```

Comment:
> ~~Thankfully we've come a long way since then, and developers take great care with accessibility and ensuring their SPAs respect the web platform.~~

:x